### PR TITLE
resource list: disable < 1.0 calamari models, rename resource

### PR DESCRIPTION
**Pull request by kba**

### CODE REVIEW

**Comment by mikegerber** [comment]
I.e. `gt4hist` should be `gt4histocr`, see https://zenodo.org/record/1344132

File: ocrd/ocrd/resource_list.yml [hunk 1]
```diff
@@ -46,16 +46,17 @@ ocrd-tesserocr-recognize:
...
-    type: tarball
-    name: qurator-gt4hist-0.3
-    description: Calamari model trained with GT4HistOCR
-    size: 116439072
-    path_in_archive: '.'
-    version_range: '< 1.0.0'
+  # XXX disabled since older ocrd_calamari versions don't support resource resolving
+  #- url: https://qurator-data.de/calamari-models/GT4HistOCR/2019-07-22T15_49+0200/model.tar.xz
+  #  type: tarball
+  #  name: qurator-gt4hist-0.3
```
Comment (mikegerber) [inline ocrd/ocrd/resource_list.yml:52]: Maybe also rename to gt4histocr in the commented part, too (prophylactically)?

**Review by mikegerber — CHANGES_REQUESTED** [review]
LGTM mostly

File: ocrd/ocrd/resource_list.yml (hunk 1 above)
Comment (kba) [reply ocrd/ocrd/resource_list.yml]: done 31ab2f8

**Review by mikegerber — APPROVED** [review]
LGTM

**Pull request merged**